rclcpp/include/rclcpp/node.hpp

```diff
@@ -163,15 +163,21 @@ class Node
     typename ServiceT,
     typename FunctorT,
     typename std::enable_if<
-      function_traits<FunctorT>::arity == 2 &&
+      function_traits<FunctorT>::arity == 2
+      >::type * = nullptr,
+    typename std::enable_if<
       std::is_same<
         typename function_traits<FunctorT>::template argument_type<0>,
         typename std::shared_ptr<typename ServiceT::Request>
-        >::value &&
+        >::value
+      >::type * = nullptr,
+    typename std::enable_if<
       std::is_same<
         typename function_traits<FunctorT>::template argument_type<1>,
         typename std::shared_ptr<typename ServiceT::Response>
-        >::value>::type * = nullptr>
+        >::value
+      >::type * = nullptr
+    >
```

Comment:
The indention of the trailing `>` was very confusing to me since they do not close on the column in which they originate. I had to trace back over all the pairs to figure out what the grouping was. Is it really our style to do this:

```c++
template<
  ...
  >
```

Rather than:

```c++
template<
  ...
>
```